.github/workflows/pr-codespell.yml

```diff
@@ -0,0 +1,19 @@
+name: Pull Request Codespell Check
+on: [pull_request]
+jobs:
+
+  codespell:
+    name: Run Codespell
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Check out the code
+      uses: actions/checkout@v2
+
+    - name: Codespell
+      uses: codespell-project/actions-codespell@master
+      with:
+        skip: .git,*.png,*.jpg,*.woff,*.ttf,*.gif,*.ico
```

Comment:
Not sure if we want to include other directors to skip here.
Considering that we have a clean run at main, I am OK having this as is.